release/api/v1alpha1/bundle_types.go

```diff
@@ -66,6 +66,7 @@ type VersionsBundle struct {
 	ControlPlane           KubeadmControlPlaneBundle   `json:"controlPlane"`
 	Aws                    AwsBundle                   `json:"aws"`
 	VSphere                VSphereBundle               `json:"vSphere"`
+	CloudStack             CloudStackBundle            `json:"cloudStack"`
```

Comment:
```suggestion
	CloudStack             CloudStackBundle            `json:"cloudStack,omitempty"`
```
New fields should be optional to keep backwards compatibility :)